elements/rh-alert/rh-alert.css

```diff
@@ -52,6 +53,7 @@
   display: grid;
   grid-template-columns: min-content 1fr;
   gap: var(--rh-space-xs, 4px);
+  font-family: var(--_font-family);
 }
```

Comment:
Border width for the container should be --rh-border-width-md for all variants

Reply:
Border width has been updated ✅